### ISSUE TITLE
Heyy.. when will app be launched?

### ISSUE THREAD

**Issue by hpk22** [tirrth/rn-cowin-bot#1]

**Comment by tirrth** [comment]
In a week or two. It's an android specific app though, Can't work it on IOS.

**Comment by hpk22** [comment]
Can't wait for it already!
Will it be completely automated booking app?

**Comment by tirrth** [comment]
Yes. You'll first have to give some insights about slot booking. If you do not choose any preferences, it'll book the first nearest slot available. Furthermore, the app will be running as a background service. As a result, even if you close the app from the recent tab, it'll still be running in the background. Btw, by default, it'll check the availability of the slots every 5-seconds and can be changed later by the user. Feel free to give any suggestions...

**Comment by hpk22** [comment]
Yes..I am somewhat hoping that app has all options of preferences like slot time,fee type, vaccine type and date,dose type,etc.
For ref you could see his repo:
https://github.com/1itromio/cowin-vaccine-booking
But I think this guy then stop working on updates.
